src/topic_training/CMakeLists.txt

```
@@ -0,0 +1,223 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(topic_training)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
```

Should tweak up to C++20 since that's what we have available.

Will they use it?  Up to them.
```suggestion
set(CMAKE_CXX_STANDARD 20)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
set(CMAKE_CXX_EXTENSIONS OFF)
```

---

Fixed in 3979e9f.